docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/StochasticDualCoordinateAscent.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification
```

Comment:
>Microsoft.ML.Samples.Dynamic [](start = 10, length = 28)

let's keep the namespace for all samples Microsoft.ML.Samples.Dynamic. The nesting is unecessary

Reply:
I've explained the reasoning here:
[https://github.com/dotnet/machinelearning/pull/2729#discussion_r260427360](https://github.com/dotnet/machinelearning/pull/2729#discussion_r260427360)


---
In reply to: [261050465](https://github.com/dotnet/machinelearning/pull/2771#discussion_r261050465) [](ancestors = 261050465)